src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -88,6 +88,40 @@ def test_self_cal(session):
     session.self_cal()
 
 
+def test_channels_rep_cap():
+    with nifgen.Session('', '', False, 'Simulate=1, DriverSetup=Model:5433 (2CH);BoardType:PXIe') as session:
+        session.func_amplitude = 0.5
+        assert session.channels['0-1'].func_amplitude == 0.5
+
+        session.channels[0].func_amplitude = 1
+        assert session.channels['0'].func_amplitude == 1
+        assert session.channels[1].func_amplitude == 0.5
+
+
+def test_markers_rep_cap(session):
+    assert '' == session.markers['Marker0'].marker_event_output_terminal
+
+    requested_terminal_name = '/Dev1/PXI_Trig0'
+    session.markers['Marker0'].marker_event_output_terminal = requested_terminal_name
+    assert requested_terminal_name == session.markers[0].marker_event_output_terminal
```

Comment:
Should I just use verbose strings everywhere (as an example for how good code should be written) in lieu of coverage for more inputs to the repeated capability?
If so, I'll apply the answer to all of the new tests.

Reply:
Do you mean `["Marker0"]` instead of `[0]`.
No, as a matter of fact, the more pythonic thing is to pass the number or an iterable, as opposed to a fully spelled out string.

I am of the opinion that the spelled-out string shouldn't even be documented behavior.

Reply:
Hmm ... we're definitely spelling out our rep_caps in the nidigital rep_caps tests, which is why I asked.
https://github.com/ni/nimi-python/blob/master/src/nidigital/system_tests/test_system_nidigital.py
Maybe it's different though, because those aren't numbers.

Yeah, if we don't document the spelled-out string, then it should be fine to only test passing the number.
I'll apply this answer to all of the tests.

Reply:
Updated all of the new tests to use integers and iterables, instead of strings. I even got rid of the '0-1' channel string.

Reply:
Well, I think we need to be a bit careful here.
Our documentation and examples should teach customers to use the API in a "pythonic" way.
But if using strings work, we should have test coverage to ensure we don't accidentally break it. It should be tested _in addition_ to the recommended usage.

Reply:
All this said, we have unit tests that test the conversion for repeated capability into strings that the driver accepts, so I don't think we need to test every permutation in system tests. This is testing what we care about which is that the correct prefix is used.